### DISCUSSION TITLE
formily2.0发展的一些想法

### DISCUSSION THREAD

**Discussion by zb188** [alibaba/formily#2460] [Ideas]
经过1周的学习，感觉formily2.0的架构思想非常棒，感谢 @janryWang 付出
目前企业用低代码实现需求越来越多
希望formily目标发展为：**企业实现低代码搭建最好的基础设施平台**
跟据官方的分层架构：[官方架构图](https://v2.formilyjs.org/zh-CN/guide#%E5%88%86%E5%B1%82%E6%9E%B6%E6%9E%84)

我的希望在：ui桥接层、组件扩展层、设计器搭建层，能提供更多的文档和实例

在引用formily2.0核心包的情况下，企业快速在ui桥接层、组件扩展层、设计器搭建层实现定制化开发

**Comment by kaoding** [comment]
formily2.0在正式发布之前的目标应该不会改：
![image](https://user-images.githubusercontent.com/18542211/142559029-eb40f1af-7e9c-4c43-a0a5-c662ce5d0a20.png)
**希望formily目标发展为：企业实现低代码搭建最好的基础设施平台** 这个可能是 formily下一个目标，至少不是 formily2.0。
再好的东西内容和目标还是要匹配的，要不然就是空口号，没有意义。

**Comment by kaoding** [comment]
文档这块到是写的不错，只是比较散，不好找，希望可以做一个”最佳实践“或者”常见问题“，罗列或者汇总一下（做一个文档导航），帮助阅读者更方便，降低心智认识，提高亲切感！

**Reply by janryWang** [reply]
可以一起共建文档的哈
